messages/setup.json

```diff
@@ -22,13 +22,13 @@
     "android:reqs:androidhome:unfulfilledMessage" : "ANDROID_HOME not found. Setup ANDROID_HOME in your environment.",
     "android:reqs:sdktools:title" : "Checking SDK Tools",
     "android:reqs:sdktools:fulfilledMessage" : "Android SDK tools were detected at %s.",
-    "android:reqs:sdktools:unfulfilledMessage" : "Install SDK tools from the range %s - %s.",
+    "android:reqs:sdktools:unfulfilledMessage" : "No supported Android SDK tools found. Minimum supported SDK tools version is %s.",
     "android:reqs:platformtools:title" : "Checking SDK Platform Tools",
     "android:reqs:platformtools:fulfilledMessage" : "Android Platform tools were detected at %s.",
-    "android:reqs:platformtools:unfulfilledMessage" : "Install at least one Android Platform tools package (%s - %s).",
+    "android:reqs:platformtools:unfulfilledMessage" : "No supported Android platform tools found. Minimum supported Android platform tools version is %s.",
```

Comment:
Heh, I didn't realize we were tying the platform tools version to the SDK version. Those two things have historically versioned roughly in the same vicinity, but ultimately independently. It's probably a safer bet with this implementation than it was previously though. 🙂